SPECS/grub2/grub2.spec

```diff
@@ -214,18 +127,16 @@ Requires:       %{name}-configuration = %{version}-%{release}
 Minimal set of utilities to configure a grub-based system
 
 %prep
-# Remove module_info.ld script due to error "grub2-install: error: Decompressor is too big"
-LDFLAGS="`echo " %{build_ldflags} " | sed 's#-Wl,-dT,%{_topdir}/BUILD/module_info.ld##'`"
-export LDFLAGS
 
-%autosetup -p1 -n grub-2.06
+%autosetup -p1 -n grub-%{version}
 cp %{SOURCE1} gnulib-%{gnulibversion}.tar.gz
 tar -zxf gnulib-%{gnulibversion}.tar.gz
 mv gnulib-%{gnulibversion} gnulib
 
 %build
-# Remove module_info.ld script due to error "grub2-install: error: Decompressor is too big"
-LDFLAGS="`echo " %{build_ldflags} " | sed 's#-Wl,-dT,%{_topdir}/BUILD/module_info.ld##'`"
```

Comment:
This appears to still be needed. You can remove it by adding:
`%undefine distro_module_ldflags`